opts/opts_test.go

```diff
@@ -113,11 +114,11 @@ func TestValidateExtraHosts(t *testing.T) {
 		`ipv6local:::1`,
 	}
 
-	invalid := []string{
-		`myhost:192.notanipaddress.1`,
-		`thathost-nosemicolon10.0.0.1`,
-		`anipv6host:::::1`,
-		`ipv6local:::0::`,
+	invalid := [][]string{
```

Comment:
Maybe a `map[string]string` would make the code a bit simpler? Loop below would become `for extraHost, expectedError := range invalid {`.